### PR TITLE
Add command to display a specified week's LOs in BrowserPanel UI

### PR DESCRIPTION
**Pull request by kohyeeru**
To act as a base for where future enhancement with Google Tasks API can be implemented, i.e. to load the week's tasks with API instead of showing module website's learning outcomes.

Add command ```view INDEX``` where BrowserPanel will load the corresponding week of learning outcomes, e.g. ```view 8``` loads content from ```https://nus-cs2103-ay1718s2.github.io/website/schedule/week8/outcomes.html```. Update both user and developer guides for the new command.

### CODE REVIEW

**Comment by iNekox3** [comment]
Please leave this PR open for now; writing JUnit tests in progress.

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -852,6 +852,25 @@ _{More to be added}_
 
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
+[discrete]
+=== Use case: View contents of a specific week
+
+*MSS*
+
+1.  User requests to view a specific week
+2.  AddressBook shows content of the specified week
```
Comment (Livian1107) [inline docs/DeveloperGuide.adoc]: shall we change the name to ProgressChecker?

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -852,6 +852,25 @@ _{More to be added}_
...
++
+Use case ends.
+
+*Extensions*
+
+* 1a. Specified week does not exist.
++
+[none]
+** 1a1. AddressBook shows an error message.
```
Comment (Livian1107) [inline docs/DeveloperGuide.adoc]: name?

**Review by Livian1107 — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (kohyeeru) [reply docs/DeveloperGuide.adoc]: Ok, will amend.

**Review by Livian1107 — APPROVED** [review]

**Pull request merged**